position/liquidity_amounts.gno

```diff
@@ -1,49 +1,53 @@
 package position
 
-func liquidityAmountsGetLiquidityForAmount0(
-	sqrtRatioAX96 bigint,
-	sqrtRatioBX96 bigint,
-	amount0 bigint,
-) bigint {
-	if sqrtRatioAX96 > sqrtRatioBX96 {
-		sqrtRatioAX96, sqrtRatioBX96 = sqrtRatioBX96, sqrtRatioAX96
+import (
+	"gno.land/r/demo/consts"
+)
+
+// swapIfGreater checkes if the first value is greater than
```

Comment:
`swapIfGreator` sounds like it `swap` for token swap, not variable swap.
any better word that we cna use?